### DISCUSSION TITLE
Application to a large container ship

### DISCUSSION THREAD

**Discussion by rpipeau3d** [capytaine/capytaine#583] [Show and tell]
Hello,
I used Capytaine to calculate the forces and movements induced by waves on a large container ship. In order to do this, I created an STL format file of the ship I was obliged to decimate, as the file was too large to be used with my small capacity computer.
A Python module was then created to obtain the different hydrostatic and hydrodynamic characteristics as well as the RAOs. As I am more familiar with the PDSTRIP program, I did some comparison between the both results (added masses, radiation damping, RAOs).
I have written also a specific module for the calculation of the drift forces according to the Far Field formulation with implementation of the Delhommeau formula. Comparison between the results of the both programs reveals notable differences.
If you find this work interesting, all the necessary files (python, stl, txt) could be downloaded here:
https://github.com/rpipeau3d/Sim.B.A.D/tree/Prime/Capytaine
Thank you

**Comment by mancellin** [comment]
Thank you for sharing this.
Far field mean drift forces are on Capytaine roadmap, so your experience with them will be useful.
